cypress/fixtures/profile/profile.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "QA",
+  "lastName": "Bot",
+  "email": "qa.tools@dibtravel.com",
+  "birthDay": "05",
```

Comment:
Revisit `number` vs. `string`. I'm fine with both, just match with data types we're sending on API